xwiki-platform-distribution/xwiki-platform-distribution-ui/xwiki-platform-distribution-ui-base/src/main/resources/XWiki/XWikiPreferences.xml

```diff
@@ -2815,6 +2815,10 @@ Your account on ${wikiname} has been successfully activated. You can now login a
 &lt;meta name="keywords" content="$escapetool.xml($keywords.replaceAll('[|,]', ' ').trim())" /&gt;
 &lt;meta name="rating" content="General" /&gt;
 &lt;meta name="author" content="$escapetool.xml($!xwiki.getUserName($doc.getAuthor(), false))" /&gt;
+## Render extensions adding content to the page head, if any.
```

Comment:
Suggestion: 

"Add HTML tags inside the page's <HEAD> element, provided by UI Extensions implementing the "org.xwiki.platform.head" Extension Point"

Reply:
Thanks :-)